src/aluno/CadastroAlunoPage.tsx

```
@@ -0,0 +1,312 @@
+        if (process.env.NODE_ENV === 'development') {
+            localStorage.setItem('validatedEmail', 'aluno.teste@utfpr.edu.br')
+        }
+    }, [])
+
+    const validatedEmail = location.state?.validatedEmail ||
+        (process.env.NODE_ENV === 'development' ? localStorage.getItem('validatedEmail') : null)
+    console.log(validatedEmail)
+
+
+    // Esquema de validação com zod
+    const formSchema = z.object({
```

ESLint

[error] Delete `··`

(prettier/prettier)